expressServer/config/strategies.js

```diff
@@ -0,0 +1,70 @@
+import { Strategy as LocalStrategy } from "passport-local";
+import { Strategy as TwitterStrategy } from "passport-twitter";
+import { Strategy as FacebookStrategy } from "passport-facebook";
+import { Strategy as GoogleStrategy } from "passport-google-oauth20";
+import config from "./";
+import { response } from "../bin";
+
+const { consumerKey, consumerSecret, callbackURL: callbackURLTwitter} = config.strategies.twitter;
+const { clientID, clientSecret, callbackURL: callbackURLFb} = config.strategies.facebook;
+const { clientID: googleID, clientSecret: googleSecret, callbackURL: callbackURLGoogle} = config.strategies.google;
+
+export default function(passport) {
+  passport.serializeUser((user, done) => done(null, user));
+
+  passport.deserializeUser((user, done) => done(null, user));
+
+  passport.use('local', new LocalStrategy({
```

Comment:
Split this file into 3 separate per each strategy (this file should only play a role of facade)